accounts/client.go

```diff
@@ -85,6 +124,7 @@ func (c *Client) UpdateEntity(entityId string, request OnboardEntityRequest) (*O
 	return &response, nil
 }
 
+// Deprecated: Use CreatePaymentInstrumentIdResponse for PaymentInstrumentRequest instead.
 func (c *Client) CreatePaymentInstrument(entityId string, request PaymentInstrument) (*common.MetadataResponse, error) {
```

Comment:
Since we are still on beta, let's call the deprecated function as `CreatePaymentInstruments` and the new one as `CreatePaymentInstrument`

Reply:
👍